contentcuration/contentcuration/views/zip.py

```diff
@@ -99,4 +117,9 @@ def get(self, request, zipped_filename, embedded_filepath):
         # allow all origins so that content can be read from within zips within sandboxed iframes
         response["Access-Control-Allow-Origin"] = "*"
 
+        # restrict CSP to only allow resources to be loaded from the Studio host, to prevent info leakage
+        # (e.g. via passing user info out as GET parameters to an attacker's server), or inadvertent data usage
```

Comment:
In this case (on Studio), don't know that `inadvertent data usage` will be as big an issue. :P -- but I guess it does prevent ZIPs from working that if later deployed to an older Kolibri version would lead to inadvertent data usage. :)